### ISSUE TITLE
ComfyUI EasyAnimateV5 text to image node missing progress

### ISSUE THREAD

**Issue by btaens** [aigc-apps/EasyAnimate#150]
In Comfyui, the image to video, control, and video to video nodes all seem to have a progress bar. 

The text to image node though is missing any kind of progress report.

**Comment by yunkchen** [comment]
Which ComfyUI workflow you using?
We have not upload formal text-to-image node yet.

**Comment by btaens** [comment]
Fair point, I did modify "Text to video generation" from here: https://github.com/aigc-apps/EasyAnimate/blob/main/comfyui/README.md

As the descriptions in other parts of the repo mention the EasyAnimateV5-12b-zh model as the text to image one, and EasyAnimateV5-12b-zh-InP as the image-to-video and video-to-video one, and it's also linked in the aforementioned readme, but the base one wasn't used in any of the ComfyUI examples, so I couldn't decide whether it using the InP model was a mistake or not. After some tests it also seemed to work better for text to image than the InP one.

The workflow I used:
[easyanimatetxt2vid.json](https://github.com/user-attachments/files/17822451/easyanimatetxt2vid.json)

Works fine honestly, other than the lack of progress while generating.

**Comment by btaens** [comment]
Oh, also, sorry, I meant text to video, not text to image.

**Comment by yunkchen** [comment]
https://github.com/aigc-apps/EasyAnimate/blob/main/comfyui/README.md#text-to-video-generation
Use our workflow, and I make sure it has progress bar.
